spec/acceptance/listen_spec.rb

```diff
@@ -7,6 +7,15 @@
   let(:all_options) { base_options.merge(polling_options).merge(options) }
 
   let(:wrapper) { setup_listener(all_options, :track_changes) }
+  around do |example|
+    warn_level = $VERBOSE
+    begin
+      $VERBOSE = nil
+      example.run
+    ensure
+      $VERBOSE = warn_level
+    end
+  end
```

Comment:
I find this a bit ugly but... otherwise on OS X 10.6 to 10.8 the spec output is full of warnings. Maybe there's a nicer way, like defaulting to calling rspec with RUBYOPT='-W0'... not sure if there are warnings you'd rather not suppress though; input appreciated.